IDEAS.md

```diff
@@ -135,20 +139,37 @@ Bridging (linking existing networks into the wider Matrix ecosystem) is one of t
   * RocketChat: https://github.com/matrix-org/matrix-appservice-rocketchat
   * Verto (FreeSWITCH): https://github.com/matrix-org/matrix-appservice-verto
  * Alpha grade:
-  * Telegram https://github.com/matrix-org/matrix-appservice-tg, https://github.com/SijmenSchoon/telematrix, https://github.com/tulir/mautrix-telegram/
   * Libpurple https://github.com/matrix-org/node-purple/tree/master/appservice
   * Asterisk https://github.com/matrix-org/matrix-appservice-respoke
- * And lots of experimental ones from the community: Twitter, XMPP, iMessage, Facebook Messenger, etc.
- * ...and the full list at https://matrix.org/blog/try-matrix-now under Application Services.
+  
+There are also a large number of community bridges, for example:
+
+  * Telegram https://github.com/SijmenSchoon/telematrix, https://github.com/tulir/mautrix-telegram/, https://github.com/matrix-org/matrix-appservice-tg
+  * Hangouts - https://github.com/Cadair/matrix-appservice-hangouts using the [hangups](http://hangups.readthedocs.io/) library.
+  * Discord - An alpha implementation is here: https://github.com/Half-Shot/matrix-appservice-discord
+  * XMPP - there are some promising bridges, but none are comprehensive.  https://github.com/pztrn/matrix-xmpp-bridge is the most advanced, but there's still lots ot work to go to make it perfect.
+  * see the full list at https://matrix.org/blog/try-matrix-now under Application Services.
+
+
+#### Bridge Project Suggestions
+
+There are a *lot* of potential projects in all the various matrix bridges, below
+is a few ideas to get you started. A good room to join if you are interested in
+working on bridges is
+[#bridges:matrix.org](https://matrix.to/#/#bridges:matrix.org)
+
+
+1. Add user puppeting support to the Slack and Gitter bridges, this would allow
```

Comment:
FYI, the `matrix-appservice-gitter` already has this feature. All that's missing is frontend UI in Riot or similar, to allow people to control it. There's even backend API hooks to tell the bridge what to do with it.